.github/workflows/rust.yml

```diff
@@ -246,6 +246,7 @@ jobs:
             mc cp -r /source/* localminio/data"
       - name: Run tests (excluding doctests)
         env:
+          RUST_MIN_STACK: 4194304
```

Comment:
This test has been modified in the `main` branch. It seems that it never happens stack overflow anymore.